pkg/apis/openebs.io/v1alpha1/cas_keys.go

```diff
@@ -45,6 +45,9 @@ const (
 	// OpenEBS
 	OpenEBSVersionKey CASKey = "openebs.io/version"
 
+	// OpenEBSUpgradeKey is the label key which provides upgrade state
+	OpenEBSUpgradeKey CASKey = "openebs.io/upgrade"
```

Comment:
please refer to my earlier comments.
If we intend to have a generic reconcile based label then we can rename the variable & set appropriate value

Reply:
Updated